packages/backend/src/peripherals/database/migrations/080_timestamp_column_tz.ts

```diff
@@ -0,0 +1,62 @@
+/*
+                      ====== IMPORTANT NOTICE ======
+
+DO NOT EDIT OR RENAME THIS FILE
+
+This is a migration file. Once created the file should not be renamed or edited,
+because migrations are only run once on the production server.
+
+If you find that something was incorrectly set up in the `up` function you
+should create a new migration file that fixes the issue.
+
+*/
+
+import { Knex } from 'knex'
+
+interface ColumnInfo {
+  table_catalog: string
+  table_schema: string
+  table_name: string
+  column_name: string
+  data_type: string
+}
+
+const tableWhitelist = [
+  'circulating_supplies',
+  'total_supplies',
+  'total_supplies_status',
+  'update_notifier',
+]
+
+export async function up(knex: Knex) {
+  // get all columns that are of type timestamp with time zone
+  const columns = (await knex('information_schema.columns')
+    .select(
+      'table_catalog',
+      'table_schema',
+      'table_name',
+      'column_name',
+      'data_type',
+    )
+    .where({
+      data_type: 'timestamp with time zone',
+      table_schema: 'public',
+    })
+    .whereIn('table_name', tableWhitelist)) as ColumnInfo[]
+
+  for (const column of columns) {
```

Comment:
iterate over tableWhitelist instead of dynamically getting the columns, they are named the same, there is no need to be implicit there

Reply:
Done